integration_test/workflow_tests/custom_logger.py

```diff
@@ -0,0 +1,16 @@
+import logging
+
+
+class CustomLoggerAdapter(logging.LoggerAdapter):
+    # example taken from
+    # https://stackoverflow.com/questions/39467271/cant-get-this-custom-logging-adapter-example-to-work
+    def process(self, msg, kwargs):
+        my_context = kwargs.pop('my_context', self.extra['my_context'])
+        return '[%s] %s' % (my_context, msg), kwargs
```

Comment:
I think you can simplify this to `return f"[{my_context}] msg", kwargs`
Check me on that.

Reply:
Tried and looks like it does not show the logs properly. For example, I get this after running the test where it just shows as `msg`

```
[Ingest TestOverrideStorageClassHappyPath] msg
[Ingest TestOverrideStorageClassHappyPath] msg
[Ingest TestOverrideStorageClassHappyPath] msg
```